app/controllers/overrides/registrations_controller.rb

```diff
@@ -0,0 +1,81 @@
+module Overrides
+	class RegistrationsController < DeviseTokenAuth::RegistrationsController
+	  def create
+      @resource            = resource_class.new(sign_up_params)
+      @resource.provider   = "email"
+
+      # honor devise configuration for case_insensitive_keys
+      if resource_class.case_insensitive_keys.include?(:email)
+        @resource.email = sign_up_params[:email].try :downcase
+      else
+        @resource.email = sign_up_params[:email]
+      end
+
+      # give redirect value from params priority
+      @redirect_url = params[:confirm_success_url]
+
+      # fall back to default value if provided
+      @redirect_url ||= DeviseTokenAuth.default_confirm_success_url
+
+      # success redirect url is required
+      if resource_class.devise_modules.include?(:confirmable) && !@redirect_url
+        return render_create_error_missing_confirm_success_url
+      end
+
+      # if whitelist is set, validate redirect_url against whitelist
+      if DeviseTokenAuth.redirect_whitelist
+        unless DeviseTokenAuth.redirect_whitelist.include?(@redirect_url)
+          return render_create_error_redirect_url_not_allowed
+        end
```

Comment:
Decat cateva linii in metoda de create sunt modificate. Gasisem o solutie pentru Devise dar devise_token_auth nu are unele metode. O sa mai verific si te anunt.